src/store/store.js

```diff
@@ -49,6 +49,15 @@ class Store {
   dump = () => {
     return cloneDeep(this._store);
   }
+
+  request = (endpoint, updatedCallback) => {
```

Comment:
In order to be consistent with regular subscriptions and to provide more flexibility it should take and pass `paramsFunc` to the subscription. Or it should take `params` and pass `() => params` if it's a one time call.